### PR TITLE
[MediaPlayer] add API to set audio-only mode

### PR DESCRIPTION
**Pull request by kqjy777**
### Description of Change ###

add API to set audio-only mode

### Bugs Fixed ###

- Provide links to bugs here

### API Changes ###

If you have the ACR for changing APIs, put the link of the ACR:
 - ACR: http://suprem.sec.samsung.net/jira/browse/TCSACR-138 

Added:
 - IsAudioOnly { get; set;} // Property

### Behavioral Changes ###

N/A

### CODE REVIEW

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs [hunk 1]
```diff
@@ -413,5 +413,32 @@ public float Volume
                     ThrowIfFailed(this, "Failed to set the volume of the player");
             }
         }
+
+        /// <summary>
+        /// Gets or sets the audio-only state.
+        /// </summary>
+        /// <value>true if the playback is audio-only mode; otherwise, false. The default value is false.</value>
+        /// The <see cref="Player"/> To set or get the player must be in the <see cref="PlayerState.Ready"/>,
```
Comment (hsgwon) [inline src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: How about remove 'To set or get the player'?

Comment (kqjy777) [reply src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: other APIs are describing like that. I will add comma to clarify the meaning

File: src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs [hunk 2]
```diff
@@ -413,5 +413,32 @@ public float Volume
                     ThrowIfFailed(this, "Failed to set the volume of the player");
             }
         }
+
+        /// <summary>
+        /// Gets or sets the audio-only state.
+        /// </summary>
+        /// <value>true if the playback is audio-only mode; otherwise, false. The default value is false.</value>
+        /// The <see cref="Player"/> To set or get, the player must be in the <see cref="PlayerState.Ready"/>,
```
Comment (hsgwon) [inline src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: How about below?
/// The `<see cref="Player"/>` must be in th the ~~

Comment (kqjy777) [reply src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs]: ok

File: src/Tizen.Multimedia.MediaPlayer/Interop/Interop.Player.cs [hunk 3]
```diff
@@ -261,6 +261,12 @@ internal static partial class NativePlayer
         [DllImport(Libraries.Player, EntryPoint = "player_get_track_language_code")]
         internal static extern PlayerErrorCode GetTrackLanguageCode(IntPtr player, StreamType type,
             int index, out IntPtr code);
+
+        [DllImport(Libraries.Player, EntryPoint = "player_set_audio_only")]
+        internal static extern PlayerErrorCode SetAudioOnly(IntPtr player, bool audioonly);
```
Comment (hsgwon) [inline src/Tizen.Multimedia.MediaPlayer/Interop/Interop.Player.cs]: How about use camel case for parameter? - `bool audioOnly`

File: src/Tizen.Multimedia.MediaPlayer/Interop/Interop.Player.cs [hunk 4]
```diff
@@ -261,6 +261,12 @@ internal static partial class NativePlayer
...
+
+        [DllImport(Libraries.Player, EntryPoint = "player_is_audio_only")]
+        internal static extern PlayerErrorCode IsAudioOnly(IntPtr player, out bool audioonly);
```
Comment (hsgwon) [inline src/Tizen.Multimedia.MediaPlayer/Interop/Interop.Player.cs]: How about use camel case for parameter? - `bool audioOnly`

**Pull request merged**